dhis-2/dhis-services/dhis-service-analytics/src/main/java/org/hisp/dhis/analytics/outlier/data/OutlierQueryParser.java

```diff
@@ -233,12 +234,13 @@ private List<OrganisationUnit> applySecurityConstrain(
    * @param relativePeriod the {@link RelativePeriodEnum}.
```

Comment:
Please, update the Java doc, as we do not use RelativePeriodEnum anymore :
```
@param relativePeriod the period.
```
Once done, the enum `RelativePeriodEnum` can be removed from the `import` section.

Reply:
Done